04-Continous-Assurance/Readme.md

```diff
@@ -708,6 +711,48 @@ The main/dominant component of the cost is automation runtime (storage/OMS costs
      - 0.27+0.07 = $0.34/year
 
 
+[Back to top…](Readme.md#contents)
+
+## Continuous Assurance (CA) - 'ScanOnDeployment' mode
+
```

Comment:
Title can be more user friendly - CA - Trigger scan on resource deployment

Reply:
Make the title more firendly